# Skillset
## R, Unix, Shell Scripting, Awk, Python
**Bioinformatics, Biostatistics, Data Mining, Data Visualization, Next Generation Sequencing Analysis (WGS, WXS, RNA-seq, WGBS, ChIP-seq, DNase-seq, Hi-C), Nanoscale DNA Imaging, Fluorescence Microscopy, Grant Writing, Public Speaking**

# Current Position
Postdoctoral Researcher at Harvard Medical School [(Wu lab)](http://transvection.org/sneha-grandhi)

# Education
**Doctor of Philosophy, Genetics and Genome Sciences**, Case Western Reserve University (2012-2018)        
**Bachelor of Science, Genomics and Molecular Genetics**, w/ minor in Mathematics, Michigan State University (2008-2012)

# Funding
Awarded individual research grant accruing $90,741 [(NCI F31 NRSA)](https://projectreporter.nih.gov/project_info_details.cfm?aid=9290948&icde=42267630&ddparam=&ddvalue=&ddsub=&cr=1&csb=default&cs=ASC&pball=) and competitive training grant accruing $56,000  [(NIGMS CMBTP)](https://projectreporter.nih.gov/project_info_description.cfm?aid=8885837&map=y) from the National Institute of Health to fund my graduate research career.

# Background
I received my bachelor’s degree in Genomics and Molecular Genetics with a minor in Mathematics from Michigan State University in 2012. With an interest in computational genomics, I earned my doctoral degree in Genetics and Genome Sciences while working in [Dr. Thomas LaFramboise](http://mendel.gene.cwru.edu/laframboiselab/)’s lab at Case Western Reserve University.

During my graduate career, I acquired a strong skillset in big data analysis and biostatistics, which was funded by two graduate research fellowships from NIH ([NCI F31 NRSA](https://projectreporter.nih.gov/project_info_details.cfm?aid=9290948&icde=42267630&ddparam=&ddvalue=&ddsub=&cr=1&csb=default&cs=ASC&pball=) and [NIGMS CMBTP](https://projectreporter.nih.gov/project_info_description.cfm?aid=8885837&map=y)) and resulted in six peer-reviewed scientific publications, two freeware software packages, two productive collaborations, and several awards (see below).

# Peer-reviewed Publications
1. Mitochondrial genomics in the cancer cell line encyclopedia and a scoring method to effectively pair cell lines for cytoplasmic hybridization.
Grandhi, S., L., Wang, J., Grandhi, A., & LaFramboise, T. (2018). 
Mitochondrion.

2. Heteroplasmic shifts in tumor mitochondrial genomes reveal tissue-specific signals of relaxed and positive selection. 
Grandhi, S.,Bosworth, C., Maddox, W., Sensiba, C., Akhavanfard, S., Ni, Y., & LaFramboise, T. (2017).
Human Molecular Genetics, ddx172.

3. Detection and quantification of mitochondrial DNA deletions from next-generation sequence data. 
Bosworth, C. M., Grandhi, S., Gould, M. P., & LaFramboise, T. (2017). 
BMC bioinformatics, 18(12), 407.

4. Phylogenetic sequence analysis and functional studies reveal compensatory amino acid substitutions in loop 2 of human ribonucleotide reductase. 
Knappenberger, A. J., Grandhi, S., Sheth, R., Ahmad, M. F., Viswanathan, R., & Harris, M. E. (2017). 
Journal of Biological Chemistry, 292(40), 16463-16476.

5. Selected mitochondrial DNA landscapes activate the SIRT3 axis of the UPR mt to promote metastasis. 
Kenny, T. C., Hart, P., Ragazzi, M., Sersinghe, M., Chipuk, J., Sagar, M. A., Eliceiri, K.W., LaFramboise, T., Grandhi, S., Santos, J., & Riar, A. K. (2017). 
Oncogene, 36(31), 4393.

6. PCR-Free Enrichment of Mitochondrial DNA from Human Blood and Cell Lines for High Quality Next-Generation DNA Sequencing. 
•	Gould, M. P., Bosworth, C. M., McMahon, S., Grandhi, S., Grimberg, B. T., & LaFramboise, T. (2016). 
PloS one, 11(5), e0156884.ome.

# Selected Awards
- 2018 Doctoral Excellence Award (Genetics) April 2018 – Issued by CWRU School of Medicine Graduate Program Directors
- Charles J. Epstein Trainee Award for Excellence in Human Genetics Research (ASHG Semifinalist) (Oct 2016) Merit-based award issued by the American Society of Human Genetics Program Committee to top-scoring and highly competitive abstracts.
